routes/extra_data_utils.go

```diff
@@ -0,0 +1,64 @@
+package routes
+
+import (
+	"encoding/hex"
+	"fmt"
+	"github.com/deso-protocol/core/lib"
+	"strconv"
+)
+
+type ExtraDataDecoder func([]byte) string
+
+var ExtraDataKeyToDecoder = map[string]ExtraDataDecoder{
+	lib.RepostedPostHash:  hex.EncodeToString,
```

Comment:
I think we want to use lib.PkToString instead of hex.EncodeToString for post hashes and public keys.